server/appServices.js

```diff
@@ -191,17 +193,20 @@ async function watchIosVariants(namespace, kubeclient) {
  * @param {String} namespace - The namespace to watch
  * @param {*} kubeclient - kubernetes client
  */
-async function watchMobileSecurityConfigMaps(namespace, kubeclient) {
-  const mssConfigMapStream = await kubeclient.api.v1.watch.namespace(namespace).configmaps.getObjectStream();
-  mssConfigMapStream.on('data', event => {
+async function watchMobileSecurityApp(namespace, kubeclient) {
+  const mssAppStream = await kubeclient.apis[mobileSecurityServiceCRD.spec.group].v1alpha1.watch
+    .namespaces(namespace)
+    .mobilesecurityserviceapps.getObjectStream();
+
+  mssAppStream.on('data', event => {
     if (event.object && event.object.metadata.name.endsWith(MOBILE_SECURITY_SUFFIX)) {
```

Comment:
This will prevent those 409 errors appearing in the server logs:

```suggestion
if (event.object && event.object.metadata.name.endsWith(MOBILE_SECURITY_SUFFIX) && !updating) {
```